content/docs/2.7/concepts/authentication.md

```diff
@@ -176,91 +176,6 @@ spec:
 
```

Comment:
I would rename the "Authentication Parameters" header to the existing "Available authentication providers for KEDA" below so that it is the following:

> ## Available authentication providers for KEDA
> 
> Authentication parameters can be retrieved from a variety of authentication providers in KEDA: 
> 
> -list auth providers-

Reply:
Done! 👍